src/libs/ReportUtils.ts

```diff
@@ -2203,16 +2248,15 @@ function getParentNavigationSubtitle(report) {
 /**
  * Navigate to the details page of a given report
  *
- * @param {Object} report
  */
-function navigateToDetailsPage(report) {
-    const participantAccountIDs = lodashGet(report, 'participantAccountIDs', []);
+function navigateToDetailsPage(report: OnyxEntry<Report>) {
+    const participantAccountIDs = report?.participantAccountIDs ?? [];
 
     if (isOneOnOneChat(report)) {
         Navigation.navigate(ROUTES.PROFILE.getRoute(participantAccountIDs[0]));
         return;
     }
-    Navigation.navigate(ROUTES.REPORT_WITH_ID_DETAILS.getRoute(report.reportID));
+    Navigation.navigate(ROUTES.REPORT_WITH_ID_DETAILS.getRoute(report?.reportID ?? ''));
 }
```

Comment:
I think there should be a guard clause here so that app accidentally doesn't go to `/r/undefined/details`